mat/hogsvd.go

```diff
@@ -215,19 +218,22 @@ func (gsvd *HOGSVD) Values(s []float64, n int) []float64 {
 
 // VTo extracts the matrix V from the singular value decomposition, storing
 // the result in-place into dst. V is size c×c.
-// If dst is nil, a new matrix is allocated. The resulting V matrix is returned.
 //
-// VTo will panic if the receiver does not contain a successful factorization.
-func (gsvd *HOGSVD) VTo(dst *Dense) *Dense {
+// If dst is empty, VTo will resize dst to be c×c. If dst is
+// non-empty, VTo will panic if dst is not c×c. VTo will also
```

Comment:
"When dst is non-empty, VTo will panic if dst is not c×c."

Reply:
Done.